dotcom-rendering/src/web/components/Border.tsx

```diff
@@ -1,12 +1,21 @@
 import { css } from '@emotion/react';
-import { from } from '@guardian/source-foundations';
+import { ArticleSpecial } from '@guardian/libs';
+import { from, neutral } from '@guardian/source-foundations';
 import { decidePalette } from '../lib/decidePalette';
+import { transparentColour } from '../lib/transparentColour';
+
+const decideBorderColour = (format: ArticleFormat) => {
+	if (format.theme === ArticleSpecial.SpecialReportAlt)
+		return transparentColour(neutral[60], 0.3);
```

Comment:
I used [`transparentColour`](https://github.com/guardian/dotcom-rendering/blob/3ac24426c459fb8456c81b5148da6c883872189e/dotcom-rendering/src/web/lib/transparentColour.ts#L26) to achieve opacity. The result colour is `#3c3c3c4d`. Not sure if in a future PR we want to add it in the [source palette](https://github.com/guardian/source/blob/main/packages/%40guardian/source-foundations/src/colour/palette.tsx#L129).

Reply:
Done!